apps/examples/board_specific_demo/ameba_secure_storage_demo/secure_storage.c

```diff
@@ -53,8 +53,8 @@ static uint32_t SLOT1_START_ADDR = 0x27000;
 #endif
 #ifdef CONFIG_AMEBASMART_TRUSTZONE
 /* ToDo: Temporary set need changes based flash addr */
-static uint32_t SLOT0_START_ADDR = 0x200000;
-static uint32_t SLOT1_START_ADDR = 0x203000;
+static uint32_t SLOT0_START_ADDR = 0x2C000;
```

Comment:
Could you let me know what they are?

Reply:
Hi @sunghan-chang , 

This is the secure storage slot address used for the secure storage test code. 
May refer to the below image:

![image](https://github.com/Samsung/TizenRT/assets/94035727/8aa67c76-938f-4c36-a680-8a255554b8c0)

Thank You.
Best Regards,
Zhen Bei

Reply:
For security, we don't use real address for this. Do you need to use real address? How about just using slot index with security api?

Reply:
Hi @sunghan-chang,

We can make it with 2 options 
1. Add a helper function in Non-Secure
2. Add an API in Secure call via NSC / SMCCC. 
Two options also need to change on TP1x, TP1e, and TPDual.
We need time to implement it and please let us know you prefer on which option.

We suggest that you may proceed with merging this PR 1st.
If have a concern about the above change, we may remove the changes 1st.
Please take note this SStorage example is unable to enable and cannot be tested once removed the SLOT address.

Thank You.

Reply:
@zhongnuo-tang There are read / write APIs for secure storage in the SE module (os/se). I don't know what you need to implement for them. Could you let us know?

Reply:
There are 2 types of read / write APIs for secure storage.
One type is NSC, which will pass the RAW data into write API, it will encrypt it and write encrypted data with a specific format into Flash. NSC read API will read the encrypted data, decrypts it and return RAW data.
One type is non-secure API, which is used to read RAW data from Flash.
NSC API required slot index.
Non-secure API required the exact address of the slot.

These are requested example, the purpose is to verify the data in Flash is not RAW data.

Reply:
@ziyik Is this an example of non-secure API?
```
One type is non-secure API, which is used to read RAW data from Flash.
NSC API required slot index.
Non-secure API required the exact address of the slot.
```
That's why it has the exact address of the slot?
For encryption verfication?

Reply:
Yes, the exact address of the slot is used by a non-secure API.
rtl_ss_flash_read() is a non-secure API to read the RAW data from Flash, so it required the exact address.
ss_read_secure_storage() and ss_write_secure_storage() are NSC API, which links to secure API, they only required a slot number to read or write.

In this secure storage example, it first reads with NSC API to check the slot data.
Then, it reads the RAW data from the actual Flash location of the data (areas A and B).
Follow by writing data to secure storage with NSC API by slot number.
Next, read with ss_read_secure_storage() to check the slot data is correctly written.
Lastly, reads the RAW data from the actual Flash location of the data (areas A and B).

Compare the data read by both APIs, data read by a non-secure API should be encrypted, it should be different from NSC API. This proves that data in Flash is encrypted.